packages/jackett/build.sh

```diff
@@ -0,0 +1,48 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/jackett/jackett"
+TERMUX_PKG_DESCRIPTION="API Support for your favorite torrent trackers"
+TERMUX_PKG_LICENSE="GPL-2.0-or-later"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.22.1468"
+TERMUX_PKG_SRCURL="https://github.com/Jackett/Jackett/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256="19a3017691273a6505a2a64046e5459b97e080e7469d9cab2f246bc04e62a40e"
+TERMUX_PKG_BUILD_DEPENDS="aspnetcore-targeting-pack-8.0, dotnet-targeting-pack-8.0"
+TERMUX_PKG_DEPENDS="aspnetcore-runtime-8.0, dotnet-host, dotnet-runtime-8.0"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_SERVICE_SCRIPT=("jackett" "exec ${TERMUX_PREFIX}/bin/jackett --DataFolder ${TERMUX_ANDROID_HOME}/.config/jackett 2>&1")
+TERMUX_PKG_BLACKLISTED_ARCHES="arm"
+TERMUX_PKG_RM_AFTER_INSTALL="
+lib/jackett/libMono.Unix.so
```

Comment:
This seems to work fine to remove these files.
Why are we removing `lib/jackett/libMono.Unix.so` here anyway?
There's still `.dll` version of the same file, so what is this for exactly and why are we only removing the `.so` version?

Reply:
So the `.dll` as well?

Reply:
Should probably verify the package works on-device then and not remove files unless we know they are unneeded.

Reply:
The AUR package includes `libMono.Unix.so` but not `libMono.Unix.dll`.

Reply:
Right but you're running a version where you probably have some files not in the current build.
I'll try to verify this current build on-device.

Reply:
Fixed it. Needed to be `cp -r build/* "${TERMUX_PREFIX}/lib/jackett"` in the make install step.

Reply:
> erroneously installed mono runtime library should be removed

You keep claiming that but do you even know why it was put there in the first place? I already mentioned that the .so file is essential and used by the updater. But here it was intentionally disabling the updater so it will not reach DllNotFoundException error.

Reply:
So both should be removed then?

Reply:
Add a mkdir -p "${TERMUX_PREFIX}/lib/jackett"

Reply:
Yeah I got it to reproduce the failure after redownloading the docker image.
I'm not sure how `docker stop termux-package-builder; docker rm termux-package-builder` managed to leave a leftover directory in the container.

Reply:
Any removal is risking more hidden errors. I will take a look if can provide a compatible libMono.Unix.so as a global package later.

Reply:
Thanks.
I don't have any expertise with .NET internals so I thought it prudent to raise the question.